src/index.scss

```diff
@@ -4,10 +4,17 @@
 @import '~@edx/paragon/dist/Icon/Icon';
 @import '~@edx/brand/paragon/overrides';
 
+@import '~@edx/frontend-component-header/dist/index';
 @import '~@edx/frontend-component-footer/dist/footer';
 
 @import 'vendor/normalize.css';
 @import 'vendor/studio.scss';
 @import 'vendor/overrides.scss';
 
 @import './library-authoring/index.scss';
+
+// Breadcrumbs
+// todo: figure out why this var isn't working (and change it to a reasonable value)
```

Comment:
I think we figured it out, right?

Reply:
yup! removed in https://github.com/brian-smith-tcril/frontend-app-library-authoring/commit/f04845484162f73a9cbce7fa714cddeb8dd5eb9c